src/adapters/apiary-blueprint-adapter.coffee

```diff
@@ -30,6 +30,26 @@ apiaryAstToApplicationAst = (ast) ->
   return blueprintApi.Blueprint.fromJSON(plainJsObject)
 
 
+resolveUriTemplate = (resource, location) ->
+  return resource.uriTemplate if resource.uriTemplate
+
+  urlPrefixPosition = getUrlPrefixPosition(location)
+  if urlPrefixPosition
+    return resource.url.slice(urlPrefixPosition)
+  else
+    return ''
+
+
+getUrlPrefixPosition = (location) ->
```

Comment:
this function is taken from `blueprint-api.coffee` file, I neither examined what does it do exactly, neither I tried to prettify it